rllib/core/models/configs.py

```diff
@@ -478,33 +578,63 @@ class CNNEncoderConfig(ModelConfig):
         #   kernel_size=[1, 1], stride=[1, 1], bias=True,
         # )
         # Flatten()
-        # Linear(121, 256)
 
     Attributes:
         input_dims: The input dimension of the network. These must be given in the
             form of `(width, height, channels)`.
         cnn_filter_specifiers: A list of lists, where each element of an inner list
             contains elements of the form
-            `[number of channels/filters, [kernel width, kernel height], stride]` to
-            specify a convolutional layer stacked in order of the outer list.
+            `[number of channels/filters, kernel, stride, [padding=same]?]` to specify a
```

Comment:
I think usage of this is not clear. We don't state that the specifier can also be given as three values.
Can we do 
```
Either [number of channels/filters, kernel, stride] or [number of channels/filters, kernel, stride, padding] to specify a 
convolutional layer stacked in order of the outer list.
```
instead?

Reply:
And then, just state that default padding is "same" but that you can specify it yourself?
Further down, be basically say now that [2, 2, 2, "same"] end up with valid pedding without raising an error or anything.

Reply:
Great catch! This was a bug. In the `output_dims` computing code, I was assuming that if padding is NOT provided, it's "same", which is correct, but if it is provided, it's always "valid", which is wrong!
Fixed.